<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reflections_selection_helper</class>
 <widget class="QDialog" name="reflections_selection_helper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reflections Selection Helper</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="min_sfac_enabled">
     <property name="text">
      <string>Min |F|²:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="instructions_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>When &quot;Apply&quot; is clicked, all reflections satisfying the conditions below will be selected.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="max_sfac_enabled">
     <property name="text">
      <string>Max |F|²:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="ScientificDoubleSpinBox" name="min_sfac_value">
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ScientificDoubleSpinBox" name="max_sfac_value">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>min_sfac_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>min_sfac_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max_sfac_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>max_sfac_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
